cypress/integration/route_spec.js

```diff
@@ -6,15 +6,16 @@ describe('Sea route tests', function () {
     beforeEach(function () {
         // Session cookies will not be cleared before the NEXT test starts
         cy.preserveSessionOnce();
+        cy.fixture('services/schedule.json').as('service');
     });
 
 
     it('Create new route', function () {
         cy.visit('/#/routes/');
 
-        cy.get('tr').should('have.length', 1);
+        //cy.get('tr').should('have.length', 1);
```

Comment:
```suggestion

```

Reply:
I removed the line and refactored the file a bit.